README.md

```diff
@@ -170,6 +170,14 @@ run tests-trigger with `--allow`:
 Of course, you should make sure that the pull request is proper and
 doesn't execute evil code during tests.
 
+### tests-trigger with a different origin
+
+If you need to specify --repo in tests-trigger as your remote is different from
+cockpit-project/cockpit you can set a git configuration option from which
```

Comment:
Run-away sentence. Please at least add a comma before the "you", but would be better to split the sentence.